tests/test_models/test_engine/test_file_storage.py

```diff
@@ -113,3 +122,73 @@ def test_save(self):
         with open("file.json", "r") as f:
             js = f.read()
         self.assertEqual(json.loads(string), json.loads(js))
+
+    @unittest.skipIf(models.storage_t == 'db', "not testing file storage")
+    def test_get_none(self):
+        """Test that get returns None on not id or object not found"""
+        storage = FileStorage()
+        for key in storage.all():
+            obj_type, dot, obj_id = key.partition('.')
+            break
+
+        obj = storage.get('', '')
+        self.assertEqual(obj, None)
+
+        obj = storage.get('', obj_id)
+        self.assertEqual(obj, None)
+
+        obj = storage.get(obj_type, '')
+        self.assertEqual(obj, None)
+
+    @unittest.skipIf(models.storage_t == 'db', "not testing file storage")
+    def test_get_error(self):
+        """Test that get only accepts two arguments"""
+        storage = FileStorage()
+        for key in storage.all():
+            obj_type = key.split('.')[0]
+            obj_id = key.split('.')[1]
+            break
+
+        with self.assertRaises(TypeError):
+            storage.get()
+        with self.assertRaises(TypeError):
+            storage.get(obj_type)
+        with self.assertRaises(TypeError):
+            storage.get(obj_type, obj_id, 'test')
+
+    @unittest.skipIf(models.storage_t == 'db', "not testing file storage")
+    def test_get_succeed(self):
+        """Test that get works succesfully"""
+        storage = FileStorage()
+        for key in storage.all():
+            obj_type = key.split('.')[0]
+            obj_id = key.split('.')[1]
+            break
+
+        obj1 = storage.get(obj_type, obj_id)
+        obj2 = storage.all()[obj_type + '.' + obj_id]
+        self.assertEqual(obj1, obj2)
+
+    @unittest.skipIf(models.storage_t == 'db', "not testing file storage")
+    def test_count_error(self):
+        """Test that count fails correctly"""
+        storage = FileStorage()
+        for key in storage.all():
+            obj_type = key.split('.')[0]
+            obj_id = key.split('.')[1]
+            break
+
+        with self.assertRaises(TypeError):
+            storage.count(obj_type, obj_id)
+
+    @unittest.skipIf(models.storage_t == 'db', "not testing file storage")
+    def test_count_succeed(self):
+        """Test that count returns correctly"""
+        storage = FileStorage()
+        for key in storage.all():
+            obj_type = key.split('.')[0]
+            obj_id = key.split('.')[1]
+            break
+
```

Comment:
You should probably loop through and test the counting of objects of each data model type, not just one of them arbitrarily.